src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -199,11 +198,23 @@ class JesBackend extends Backend with LazyLogging {
       case (fileTag: String, location: WdlFile) =>
         logger.info(s"${makeTag(backendCall)}: $fileTag -> ${location.valueString}")
         Seq(mkInput(fileTag, location))
-      case (fileTag: String, location: WdlArray) => location.value.collect { case f: WdlFile => mkInput(fileTag, f) }
+      case (fileTag: String, localPathWdlArray: WdlArray) =>
```

Comment:
I still advocate for renaming `fileTag` to `locallyQualifiedName` just for added clarity